components/common/mojomCache.ts

```diff
@@ -3,40 +3,32 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // You can obtain one at https://mozilla.org/MPL/2.0/.
 
```

Comment:
Let's add comment at the top about the main export - and an example of what mojom should look like, as well as boilerplate in JS

Reply:
Added, but to the EntityCachingWrapper